README.md

```diff
@@ -975,13 +975,16 @@ $ node -e "import('./index.js').then(m => console.log(m.main))"
 ```
 
 Spago does not wrap the entirety of the bundler's API (esbuild for JS builds), so it's possible to pass arguments through to it. E.g. to exclude an NPM package from the bundle you can pass the `--external` flag to esbuild:
+
```

Comment:
My editor auto-formatted this markdown on save - can revert if preferred.

Reply:
Yes please, reformatting of the markdown documents (both the README and the CHANGELOG) are not relevant/desired and should be reverted